subsys/bluetooth/controller/hci/hci.c

```diff
@@ -4951,6 +4951,20 @@ static void vs_read_tx_power_level(struct net_buf *buf, struct net_buf **evt)
 }
 #endif /* CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL */
 
+#if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY)
+static void vs_set_scan_req_rcvd_mask(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_cp_vs_set_scan_req_rcvd_mask *cmd = (void *)buf->data;
+
+	if (cmd->enable != 0) {
+		le_event_mask |= BT_EVT_MASK_LE_SCAN_REQ_RECEIVED;
+	} else {
+		le_event_mask &= ~BT_EVT_MASK_LE_SCAN_REQ_RECEIVED;
+	}
```

Comment:
Set a variable that can be used to override `adv->lll.scan_req_notify` assignment.

Reply:
I've used the corresponding bit in `vs_events_mask`.